deployment/configgen/CMakeLists.txt

```diff
@@ -45,9 +45,12 @@ install ( TARGETS configgen DESTINATION sbin )
 target_link_libraries ( configgen 
         deploy 
         jlib 
-        environment 
         )
 
+if (NOT CONTAINERIZED)
+    target_link_libraries ( configgen environment )
```

Comment:
doesn't configgen depend on environment symbols, so will fail to link in containerized mode without them?
I presume it doesn't hit a build error, but wondering why..

Shouldn't we leave them as is, but instead exclude them building at all, i.e. exclude configgen, configenv etc. in containerized mode?

Reply:
You are correct - they are not built in container mode - I will regress